README.rst

```diff
@@ -1,6 +1,16 @@
 STARDIS Stellar Radiative transfer
 ----------------------------------
 
+Welcome to STARDIS!
+STARDIS is an open source LTE Python package that utlizes the TARDIS infrastructure 
```

Comment:
Bit of a weird description. LTE is describing the specifics of the radiative transfer simulation, it's not really a descriptor of "python package". You also don't have to say that it's open source. They're already looking at the github so it should be clear that the code is open. Also "utilizes" has a typo.